pkg/task/backup.go

```diff
@@ -35,6 +35,7 @@ const (
 	flagGCTTL = "gcttl"
 
 	defaultBackupConcurrency = 4
+	maxBackupConcurrency     = 1024
```

Comment:
This is too large, it will spawn 1024 thread, it could fail due to no resource (can not create threads). What about 128?